hrui/hrui.php

```diff
@@ -590,6 +590,17 @@ function hrui_civicrm_managed(&$entities) {
 }
 
 /**
+ * Implements hook_civicrm_alterMenu().
+ *
+ * @param Array $items List of http routes
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_navigationMenu
```

Comment:
I think the correct link here is this one: https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_alterMenu/